### PR TITLE
Ltp-39-contact-admin

### PR DESCRIPTION
**Pull request by Benzene06**
Need an edit to forum.php to bring users to the contact page

### CODE REVIEW

**Review by Kodoh — COMMENTED** [review]
Does it actually send the user an email?

**Comment by Benzene06** [comment]
Mail can only be sent after the websites are put onto a server

**Comment by Benzene06** [comment]
Button for forum.php:
<input type="button" onclick="contactadmin.html" value="Contact Us" />

(Not sure how to do pull request for master branch

File: contactstyle.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
+*{
```
Comment (Kodoh) [inline contactstyle.css:1]: Can this go into style.css?

File: contactadmin.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (Kodoh) [inline contactadmin.html:2]: For consistency can we make this a php?

File: contactstyle.css (hunk 1 above)
Comment (Benzene06) [reply contactstyle.css:1]: Is there a function that I need to add?

File: contactadmin.html (hunk 2 above)
Comment (Benzene06) [reply contactadmin.html:2]: Is it possible to use _action="contact.php"_ even when this becomes a php?

Comment (Benzene06) [reply contactadmin.html]: There we go

**Pull request closed without merging**